app/components/editor/elements/SourceSelector.vue

```diff
@@ -135,7 +157,7 @@ i.disabled {
 }
 
 .icon--active {
-  color: var(--teal);
+  color: var(--icon-active);
```

Comment:
Lol it's maybe slightly counterintuitive that `--teal` is set to `@goldenrod` in `themes.g.less`.  We should probably rename that variable at some point.

Reply:
yea i don't disagree-- maybe something like `--brand-primary` or `--primary-accent` would be better